io_p.rb

```diff
@@ -0,0 +1,93 @@
+require 'json'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student_s'
+require_relative 'teacher_t'
+
+module SaveData
+  def save_books
+    return unless File.exist?('./data_files/books.json')
+    return unless @books.any?
+
+    books_data = JSON.generate(@books, { max_nesting: false })
+    File.write('./data_files/books.json', books_data)
+  end
+
+  def save_people
+    return unless File.exist?('./data_file/people_p.json')
+    return unless @people.any?
+
+    people_data = JSON.generate(@people, { max_nesting: false })
+    File.write('./data_file/people-P.json', people_data)
+  end
+
+  def save_rentals
+    return unless File.exist?('./data_file/rentals.json')
+    return unless @rentals.any?
+
+    rentals_data = JSON.generate(@rentals, { max_nesting: false })
+    File.write('./data_file/rentals.json', rentals_data)
+  end
+
+  def save()
+    save_books
+    save_people
+    save_rentals
+  end
+end
```

Comment:
- [ ] [REQUIRED] Great job team 👍🏽 Unfortunately, it looks like the save module has some issues you need to look into. Whenever I exit the app, I get the error below:
<img width="977" alt="Screenshot 2023-01-12 at 14 17 29" src="https://user-images.githubusercontent.com/58692590/212077059-1409164a-8672-48a0-9ceb-3f031f626985.png">
Kindly fix this and ask for another review.

Reply:
Thanks for the feedback, we will make fix the issue as soon as possible